test/support/api_case.ex

```diff
@@ -66,4 +76,54 @@ defmodule CodeCorps.ApiCase do
     {conn, admin}
   end
 
+  defmacro define_request_helper_methods(resource_name: resource_name), do: do_add_request_helper_methods(resource_name)
+  defmacro define_request_helper_methods(_), do: nil
+
+  defp do_add_request_helper_methods(resource_name) do
```

Comment:
I'm still looking for a slightly better way to do this (instead of one huge `quote` block), but it ought to be good enough for this PR. We should create an issue to switch all of our other applicable controller tests to use this.